common/static/common/js/spec/components/paging_spec.js

```diff
@@ -73,19 +73,19 @@ define([ "jquery", "common/js/spec_helpers/ajax_helpers", "URI",
                 describe("setPage", function () {
                     it('can set the current page', function () {
                         var requests = AjaxHelpers.requests(this);
-                        pagingView.setPage(0);
+                        pagingView.setPage(1);
```

Comment:
`collection.currentPage` is the page currently represented by the collection using the server's numbering scheme. In this test it is zero indexed. Ideally, future code will not access currentPage directly but will instead go though `currentOneIndexPage()`.